txpool/txpool.go

```diff
@@ -317,6 +321,9 @@ func (p *TxPool) Pop(tx *types.Transaction) {
 	// pop the top most promoted tx
 	account.promoted.pop()
 
+	//	successfully popping an account reset its demotions count to 0
```

Comment:
Typo here, should be `resets`

Reply:
[Fixed](https://github.com/0xPolygon/polygon-edge/pull/541/commits/14064c8dbc0780a5ab75a85499bf9127fb6d608b)